catalyst_voices/packages/catalyst_voices_services/lib/src/registration/registration_service.dart

```diff
@@ -0,0 +1,107 @@
+import 'package:catalyst_cardano/catalyst_cardano.dart';
+import 'package:catalyst_cardano_serialization/catalyst_cardano_serialization.dart';
+import 'package:catalyst_voices_models/catalyst_voices_models.dart';
+import 'package:catalyst_voices_repositories/catalyst_voices_repositories.dart';
+import 'package:catalyst_voices_services/src/registration/registration_transaction_builder.dart';
+
+/// Manages the user registration.
+final class RegistrationService {
+  final TransactionConfigRepository _configRepository;
+
+  const RegistrationService(this._configRepository);
+
+  /// Returns the available cardano wallet extensions.
+  Future<List<CardanoWallet>> getCardanoWallets() {
+    return CatalystCardano.instance.getWallets();
```

Comment:
`CatalystCardano` should be injected, just like `TransactionConfigRepository` is